### ISSUE TITLE
Desativar 'Página de Contato' padrão da Rede Livre

### ISSUE THREAD

**Issue by uira** [redelivre/redelivre#104]
- [ ] Desativar Página de Contato da RedeLIvre, tirar do menu e tirar a funcionalidade.

**Justificativa:** A recorrência de bugs (#29) e as opções mais robustas de página de contato já disponíveis nos temas, no CF7 e no próprio JetPack que tá sendo proposto.

**Comment by anapaulabrasa** [comment]
Certo. É preciso levar em consideração a quantidade de projetos na redelivre que utilizam a funcionalidade. Basicamente todo projeto realmente ativo na rede, e que usa o templates próprios, usa a página de contatos.

**Comment by uira** [comment]
@anapaulabrasa Podemos olhar pra esses dados?

Conseguimos saber exatamente quantos são os projetos na Rede que Utilizam essa funcionalidade?

Pode colocar o link de alguns aqui, para que eu possa estudar a alternativa de migração?

**Comment by anapaulabrasa** [comment]
Vou buscar alguns sites, mas não sei se tem como fazer um filtro e puxar exatamente esta quantidade.

**Comment by anapaulabrasa** [comment]
Seguem alguns, @uira 

http://www.cefuria.org.br/
mesmascoisas.art.br/
http://www.pacs.org.br/
http://recid.redelivre.org.br
http://projetomidialivre.redelivre.org.br/
http://www.plataformadh.org.br/
http://www.ceeconhecimento.com.br/
http://memorialchicomendes.org/

**Comment by uira** [comment]
@anapaulabrasa @jacsonp vc consegue fazer uma consulta via banco, e olhar quais projetos preencheram os campos de configuração da página de contato?

**Comment by jacsonp** [comment]
Consigo, mas não é uma query simples, sem condições de fazer isso agora

**Comment by eduardozulian** [comment]
MInha opinião é antiga e talvez não se encaixe na redelivre no momento, mas remover essa funcionalidade dos temas e jogar para um plugin externo pode simplificar algo que era um pouco travado antes. Alguns benefícios:

* Remove a necessidade de adicionar um template específico
* Abre a opção das pessoas terem mais de um formulário por site
* Permite a criação de campos extras

Uma ideia que pode ser possível para o futuro:

1. Ativar o novo plugin (seja o CF7, o Jetpack etc) para novos sites ou mantê-lo oculto para sites criados antes de uma certa data limite (de instalação do novo form);
2. Desabilitar a Página de Contato padrão para novos sites, forçando o usuário a usar o novo plugin
3. Site por site, ajudar na migração para a nova funcionalidade
